lntest/itest/lnd_test.go

```diff
@@ -13,39 +13,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const (
-	// defaultSplitTranches is the default number of tranches we split the
-	// test cases into.
-	defaultSplitTranches uint = 1
-
-	// defaultRunTranche is the default index of the test cases tranche that
-	// we run.
-	defaultRunTranche uint = 0
-)
-
 var (
-	// testCasesSplitParts is the number of tranches the test cases should
-	// be split into. By default this is set to 1, so no splitting happens.
-	// If this value is increased, then the -runtranche flag must be
-	// specified as well to indicate which part should be run in the current
-	// invocation.
-	testCasesSplitTranches = flag.Uint(
```

Comment:
Aren't these still need to run the old tests?

Reply:
They are moved to `temp_lnd_test.go`. It's done this way so we can later on just remove `lnd_test.go` file and rename `temp_lnd_test.go -> lnd_test.go`.